crates/rspack_binding_builder_testing/package.json

```diff
@@ -11,17 +11,17 @@
     "binding.d.ts"
   ],
   "scripts": {
-    "build:dev": "node scripts/build.js",
-    "build:debug": "node scripts/build.js --profile release-debug",
-    "build:ci": "node scripts/build.js --profile ci",
-    "build:profiling": "node scripts/build.js --profile profiling",
+    "build:dev": "rspack-builder",
```

Comment:
if all the logic moved to binding-builder-cli why the testing's script/build.js is removed?

Reply:
Done